ads/4wppn.md

```diff
@@ -0,0 +1,43 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# 4w Premium Publisher Network
+
+Please refer to [Help Desk](4winfo@4wmarketplace.com) for more
+information on how to get required ad tag or placement IDs.
+
+## Examples
+
+### Simple ad tag 
+
+```html
+ <amp-ad width=320 height=250
+          type="adagio"
```

Comment:
Typo?

But is there any chance you could actually do with a single ad type for all of these and switch inside the implementation based on some param?